src/Core/Snippets/Snippets.cs

```diff
@@ -145,7 +144,11 @@ public Snippet Compile(string code)
             if (string.IsNullOrWhiteSpace(code)) throw new ArgumentNullException(nameof(code));
 
             var duration = Stopwatch.StartNew();
-            var logger = new QSharpLogger(Logger);
+            var errorCodesToIgnore = new List<string>()
+            {
+                "QS6241",   // Invalid entry point. Only executable Q# projects can have entry points.
```

Comment:
Is this a short term solution (ignoring the error) and if so, what's the long term?

Reply:
This is intended to be the long-term solution. Having an `@EntryPoint()` attribute in a notebook cell is somewhat meaningless (since we will automatically generate an entry point whenever trying to execute one of these operations), and we don't want to bother the user with an error related to this when we can just as easily ignore it.